src/utils/string.js

```diff
@@ -16,7 +16,21 @@ function removeTrailingPeriod(str) {
 	return str[str.length - 1] === "." ? str.substring(0, str.length - 1) : str;
 }
 
+/**
+ * Strips ansi escape codes
+ * @param {string} str - Console output to escape
+ * @returns {string} - Pure ansi output
+ */
+function removeANSIColorCodes(str) {
```

Comment:
I had trouble even with `-no-colors` with unit tests locally. Perhaps some limitation of the command line tool?